specs/tasks.js

```diff
@@ -91,7 +91,30 @@ function processAction(action) {
       browser.frame(action.switchToFrame)
     }
 
-    if(action.wait) {
+    if (action.dragAndDrop !== undefined){
+
+      browser.execute(function(elemXpath, offsetX, offsetY) {
```

Comment:
Could you please move the anonymous function inside `browser.execute` to a named one?